src/EntityServer.ts

```diff
@@ -44,6 +45,7 @@ import AssignmentClient from "./domain/AssignmentClient";
  *
  *  @property {number} maxOctreePacketsPerSecond - The maximum number of octree packets per second that the user client is
  *      willing to handle.
+ *  @property {Signal} addedEntity - Triggered when an entity data packet is received.
```

Comment:
```suggestion
 *  @property {Signal<EntityServer~AddedEntityCallback>} addedEntity - Triggered when an entity data packet is received.
```
See the `MessageMixer.messageReceived` JSDoc as an example.

Reply:
Fixed.